features/steps/machines.py

```diff
@@ -110,6 +110,9 @@ def given_a_machine(
     # make sure the machine has up-to-date apt data
     when_i_apt_update(context, machine_name=machine_name)
 
+    # add coverage
+    when_i_apt_install(context, "python3-coverage", machine_name=machine_name)
```

Comment:
I think we may want to make the whole act of collecting coverage optional.
You can add a new field to `UAClientBehaveConfig` and then check it throughout like so:
```suggestion
    if context.pro_config.collect_coverage:
        when_i_apt_install(context, "python3-coverage", machine_name=machine_name)
```

Reply:
Yes, that's a great idea